front/src/applications/operationalStudies/views/ImportTimetableItem.tsx

```diff
@@ -29,10 +30,7 @@ const ImportTimetableItem = ({ scenario, upsertTimetableItems }: ImportTimetable
     paced_trains: [],
   });
 
-  const { data: { results: rollingStocks } = { results: [] }, isError } =
-    osrdEditoastApi.endpoints.getLightRollingStock.useQuery({
-      pageSize: 1000,
-    });
+  const { rollingStocks, isError } = useRollingStockContext();
 
   useEffect(() => {
```

Comment:
I'm not a huge fan to return `isError` here, because each caller needs extra code to handle the error (or they just ignore it). Could we move the `setFailure()` call used in this `useEffect()` inside `RollingStockContextProvider`? That way, error handling is centralized.